project hackyourweather/hackyourweather/src/Components/CityBox/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './styles.css'
+
+
+
+function CityBox(props) {
+    const city = props.data
+
+    return (
+        <div>
+            {city.cod === 200 && < div className="cityBox found" >
```

Comment:
Ternary if would be more suitable
```suggestion
            {city.cod === 200 ? < div className="cityBox found"> .... : <div className="notFound">
```